docs/src/reference/orientation-files.rst

```diff
@@ -0,0 +1,56 @@
+Overview of this repository (i.e., what do these files do?)
+===========================================================
+
+The files in this repository fall into one of these categories:
+
+.. contents::
+   :local:
+
+Default workflow configuration files
+------------------------------------
+
+-  ``defaults/parameters.yaml``: default workflow configuration parameters. Override these settings in the workflow configuration file used by ``--configfile``.
+-  ``defaults/auspice_config.json``: default visualization configuration file. Override these settings using ``auspice_config``. `See the customization guide for visualizations <customizing-visualization.md>`__.
+-  ``defaults/include.txt``: list of strain names to *include* during subsampling and filtering (one strain name per line)
+-  ``defaults/exclude.txt``: list of strain names to *exclude* during subsampling and filtering (one strain name per line)
+
+Custom configuration files
+--------------------------
+
+These will be stored in ``my-analyses/``.
+
+- ``my-analyses/.../builds.yaml``: workflow configuration file used by ``--configfile`` where you can define and parameterize the builds you want to run. `See the customization guide <customizing-analysis.md>`__.
```

Comment:
Concern about removal of `my_profiles/` with the current replacement of `my-analyses/`:

A decent amount of reference material relies on `my_profiles/` or an equivalent being present. This is fine when `my_profiles/` is guaranteed to be in the repo when cloned.

However, see the above modified snippet of [orientation-files](https://docs.nextstrain.org/projects/ncov/en/latest/analysis/orientation-files.html#workflow-configuration-files-we-might-want-to-customize) which assumes that the reader has `my-analyses/`.

This reference material would only make sense for readers who have gone through the tutorial, or at least the step where they clone `ncov-tutorial` into `my-analyses/`.

Do we want to push hard for users to go through that cloning step? If so, then maybe it would be best to put that step in the setup instructions. If not, I can't think of a good alternative here.

Reply:
Yeah, removing a directory that people may expect is always a bit complicated. To smooth the path, how about:

1. A `.. hint::` in the orientation-files explaining that `my_profiles` is no longer
2. Leave a `./my_profiles/README.md` with the contents of (1). We can make a reminder to delete this in 2 (?) months.

I don't know best practices around template repos etc etc, but I did find it a bit strange that we clone `ncov-tutorial` into `my-analyses`. An alternative would be to keep that folder as the canonical tutorials and ask people to add their config files to `./my-analyses` (perhaps initialising it with a `./my-analyses/README.md`).

Reply:
Relevant discussion here: https://github.com/nextstrain/ncov/pull/894#discussion_r832683570

I think the point of ncov-tutorial as a template repo is to make it easier to build upon the tutorial files, rather than copy/pasting from another directory or starting from scratch.

Two potential options:

### 1. With template repo:

- In setup steps: direct users to make their own `my-ncov-analyses` repo from the `ncov-tutorial` template repo.
    - Not `my-analyses` since I imagine something like `https://github.com/victorlin/my-analyses` is too generic.
- Reference `my-ncov-analyses/` in all the these docs.
    - I don't think this is the right approach, it assumes people have gone through the tutorial setup. But I can't think of a better option.

### 2. Without template repo (@jameshadfield's suggestion above):

- Provide a file `ncov/my-analyses/README.md`.
    - Primary purpose is to have `my-analyses/` already available, similar to current `my_profiles/`.
- In example-data tutorial: `git clone ncov-tutorial` without renaming.
- At the end of genomic-surveillance tutorial: copy `ncov-tutorial/genomic_surveillance.yaml` to `my-analyses/genomic_surveillance.yaml`, then edit there.
    - Add note to initialize `my-analyses/` as a git repo to enable version control (recommended).
- Reference `my-analyses/` in all the these docs.

---

I still feel like this would be simpler [if `ncov/` was used as a standalone directory](https://bedfordlab.slack.com/archives/C02Q4Q5S85A/p1646334170382399). I'm playing with @huddlej's suggestion on [Modularization](https://snakemake.readthedocs.io/en/stable/snakefiles/modularization.html) to see if this experience can be improved.

Reply:
[Latest approach from Slack](https://bedfordlab.slack.com/archives/C02Q4Q5S85A/p1649794496745729?thread_ts=1649180310.784879&cid=C02Q4Q5S85A):

1. For the tutorial, we tell readers to `git clone ncov-tutorial` without renaming the directory.
2. Add a guide to create a personal repo from `ncov-tutorial` template.
    - We recommend naming the repo `my-ncov-analyses/` since it will be automatically excluded by `ncov/.gitignore`.
    - `git clone https://github.com/victorlin/my-ncov-analyses` will create an "analysis directory" `ncov/my-ncov-analyses/`.
3. For orientation-files which currently has lots of references to `my_profiles/`, we add a new section describing the "analysis directory" and move current `my_profiles/` references to this new section.

1 and 2 are done: 3644c310afbe6cec582fc6981778ee0abad3fc8f...334a0d087e3bc1ffde0784539610e7ba474a2af0

Reply:
I think this should be good with [the new `tutorial/next-steps` page](https://nextstrain--894.org.readthedocs.build/projects/ncov/en/894/tutorial/next-steps.html) + [a rewrite of the `reference/files` page](https://nextstrain--894.org.readthedocs.build/projects/ncov/en/894/reference/files.html#analysis-directory) linking to the former:

![reference-files-analysis-directory](https://user-images.githubusercontent.com/13424970/164303577-e69f52c9-7b87-4db6-b381-b237520c4a5f.png)

Reply:
> 1. A `.. hint::` in the orientation-files explaining that `my_profiles` is no longer
> 2. Leave a `./my_profiles/README.md` with the contents of (1). We can make a reminder to delete this in 2 (?) months.

Added these as suggested by @jameshadfield.